contracts/SystemReward.sol

```diff
@@ -54,4 +56,29 @@ contract SystemReward is System, ISystemReward{
   function isOperator(address addr) external view returns (bool) {
     return operators[addr];
   }
-}
\ No newline at end of file
+
+  function updateParam(string calldata key, bytes calldata value) onlyGov external override {
+    if (Memory.compareStrings(key, "addOperator")) {
```

Comment:
@Pythonberg1997  is it possible that when BC hard fork happen, it can generate a cross-chain package automatically? (so we do not have to gov again)